tools/generators/utils/linting.ts

```diff
@@ -0,0 +1,33 @@
+import { Tree, Rule } from '@angular-devkit/schematics';
```

Comment:
this file has been copied from another PR https://github.com/SAP/fundamental-ngx/pull/7441, since that one is not merged yet.